requirements/dev.txt

```diff
@@ -74,11 +68,9 @@ sphinxcontrib-jsmath==1.0.1  # via sphinx
 sphinxcontrib-qthelp==1.0.2  # via sphinx
 sphinxcontrib-serializinghtml==1.1.3  # via sphinx
 stevedore==1.31.0         # via doc8
-text-unidecode==1.3       # via faker
 toml==0.10.0              # via black
 traitlets==4.3.3          # via ipython
 typed-ast==1.4.0          # via black
-urllib3==1.25.7           # via requests
```

Comment:
I don't get why this file is ending up including requests but not dependencies of requests.

Reply:
Because that's what `lock:py` gave us 🤷‍♀

Reply:
lock:py? I don't see that in package.json

Reply:
It's a command in `derrick` that runs `pip-compile`.

Reply:
Ah yes I see. I guess as long as these removed requirements are in prod.txt it should work out okay.